internal/gapicgen/generator/gapics.go

```diff
@@ -296,13 +296,13 @@ func (g *GapicGenerator) microgen(conf *microgenConfig) error {
 		"-I", g.protoDir,
 		"--go_gapic_out", g.googleCloudDir,
 		"--go_gapic_opt", fmt.Sprintf("go-gapic-package=%s;%s", conf.importPath, conf.pkg),
-		"--go_gapic_opt", fmt.Sprintf("api-service-config=%s", conf.apiServiceConfigPath)}
+		"--go_gapic_opt", fmt.Sprintf("api-service-config=%s", filepath.Join(conf.inputDirectoryPath, conf.apiServiceConfigPath))}
 
 	if conf.releaseLevel != "" {
 		args = append(args, "--go_gapic_opt", fmt.Sprintf("release-level=%s", conf.releaseLevel))
 	}
 	if conf.gRPCServiceConfigPath != "" {
-		args = append(args, "--go_gapic_opt", fmt.Sprintf("grpc-service-config=%s", conf.gRPCServiceConfigPath))
+		args = append(args, "--go_gapic_opt", fmt.Sprintf("grpc-service-config=%s", filepath.Join(conf.inputDirectoryPath, conf.inputDirectoryPath, conf.gRPCServiceConfigPath)))
```

Comment:
Looks like `conf.inputDirectoryPath` is duplicated?

Reply:
https://github.com/googleapis/google-cloud-go/pull/4397